Card/src/main/java/com/braintreepayments/api/card/AuthenticationInsight.kt

```diff
@@ -0,0 +1,40 @@
+package com.braintreepayments.api.card
+
+import android.os.Parcelable
+import com.braintreepayments.api.sharedutils.Json
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+@Parcelize
+data class AuthenticationInsight(val regulationEnvironment: String) : Parcelable {
```

Comment:
Same here: there are some doc strings in [AuthenticationInsight.java](https://github.com/braintree/braintree_android/blob/f13626a796ae46b21fe41d62b0092c7f73d3c036/Card/src/main/java/com/braintreepayments/api/card/AuthenticationInsight.java#L48) that will need to be ported over.